front/app/components/admin/ResourceList/itemOrderWasUpdated.ts

```diff
@@ -0,0 +1,11 @@
+import { Item } from './SortableList';
+
+export default (prevItems: Item[], currentItems: Item[]) => {
```

Comment:
Wondering if you really need this and can't do the state update after the handleDropRow function

Reply:
If I remember correctly, `componentDidUpdate` would be called multiple times, and when called the first time the updated backend data would not be available yet. So then it would quickly flash back to the previous state (before the row was dropped to its new place) and then go to the correct state after the backend data came in. I guess we could make this work more cleanly with a functional component and `useEffect`